BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -125,6 +126,19 @@ class CbrainTask::<%= name %> < <%= descriptor['cbrain:inherits-from-class'] ||
   # Note that this function also generates the list of output filenames
   # in params.
   def cluster_commands #:nodoc:
+
+    # Environment variables from Boutiques descriptor
+% if eVars.empty?
```

Comment:
Indeed. However, the reason for it is a tiny prettification change in the output code, because if the array is empty, I'd prefer to see
```
envVars = []
```
rather than
```
envVars = [
]
```